### ISSUE TITLE
Text change in forgetful browsing global setting

### ISSUE THREAD

**Issue by pes10k** [brave/brave-browser#30163]
Currently the global toggle for forgetful browsing in brave://settings/shields says "Forget me when i close this site". But, since this a global toggle, there is no "this" site. Would be good to change the text to something like "Forget me when I close a site"

<img width="698" alt="Screenshot 2023-05-05 at 09 26 08" src="https://user-images.githubusercontent.com/76526/236485935-ef8854d0-e152-4438-84bf-370c0af22e82.png">

**Comment by bsclifton** [comment]
Verified it's still an issue on Desktop and Android.

Would be important to make sure the resource ID used for the settings page isn't the same one as the one in shields panel. For example, we'll want to add a new resource that says `Forget me when I close a site`.
https://github.com/brave/brave-core/blob/a89691ffa7f7bc1f75555daa25642d53bb6042e4/components/resources/brave_shields_strings.grdp#L116-L118

**Comment by anushagh** [comment]
Hey, I'm interested in working on this issue. Could you assign it to me?

**Comment by pes10k** [comment]
I don’t think we generally assign issues to folks outside of brave. But I’m sure we’d welcome contributions and fixes. And if you end up submitting a fix we merge, I expect there’d be no problem assigning the issue to you retroactively
